app/pb_kits/playbook/pb_body/_body.jsx

```diff
@@ -2,50 +2,59 @@
 
 import React from 'react'
 import classnames from 'classnames'
+import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
 import { Highlight } from '../'
 import { spacing } from '../utilities/spacing.js'
 
 type BodyProps = {
+  aria?: object,
   className?: String,
   children?: Array<React.ReactChild>,
-  color: 'light' | 'lighter',
+  color: 'dark' | 'default' | 'light' | 'lighter' | 'light_dark' | 'lighter_dark',
   dark?: Boolean,
-  status?: 'negative' | 'positive',
-  tag: String,
-  text?: String,
+  data?: object,
+  highlightedText?: Array<String>,
   highlighting?: Boolean,
-  highlightedText?: Array<String>
-}
-
-const bodyCSS = ({
-  color = '',
-  dark = false,
-  status = '',
-
-}: BodyProps) => {
-  const colorStyle = color !== '' ? `_${color}` : ''
-
-  const themeStyle = dark === true ? '_dark' : ''
-
-  const statusStyle = status !== '' ? `_${status}` : ''
-
-  return 'pb_body_kit' + colorStyle + themeStyle + statusStyle
+  id?: String,
+  status?: 'negative' | 'neutral' | 'positive',
+  tag?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'span' | 'div',
```

Comment:
This will possibly break things in Nitro. It's a possible breaking change if any Body kit is currently using a tag not in this list. I think let's hold off on releasing this and take some time to go through the 109 occurences of this kit in Nitro, and double-check that none of those instances are using the tag prop in a manner excluded from this list of options. Here is the link to all the occurrences that we will need to look at for regression: https://github.com/powerhome/nitro-web/search?l=JSX&q=%3CBody+tag%3D%22a%22

Let me know if there are further questions. I'm happy to pair on this. It will probably take a half hour at least to go through and make sure. @kellyeryan cc: @thestephenmarshall @jasoncypret

Reply:
@jasperfurniss AFAIK these are merely suggestions and won't impact anything in Nitro (unless something there has significantly changed that I am unaware of). Technically, someone could send `iframe` and it would attempt to do that since there really isn't any true type guardian, but rather Flow is an advisor. I would love to be proven wrong of course.

Reply:
@thestephenmarshall You're right! I was mixing this one up with the .rb file changes made  previously.